src/fhirtypes/ElementDefinition.ts

```diff
@@ -472,6 +473,56 @@ export class ElementDefinition {
     this._original = undefined;
   }
 
+  clearOriginalProperty(pathParts: PathPart[]): void {
+    let currentOriginalElement: any = this._original;
+    // eslint-disable-next-line
+    let currentActualElement: any = this;
```

Comment:
I just want to double check my own understanding of why we need to keep track of both `this` and `this._original`. Is it just so that `this._original` has the same starting point for type.aggregation so when the diff is generated it is correct?

Reply:
Exactly. We need to clear out something on `this._original`, but we have to follow the `clearPath` to get there. Then, when the diff is generated, that part of `this._original` will be `undefined`, which means that the diff will be everything at that path on `this`.

Reply:
Thanks for confirming!